.github/workflows/deploy.yml

```diff
@@ -0,0 +1,53 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+
+jobs:
+  canary:
+    runs-on: ubuntu-latest
+    if: github.ref_name == 'develop'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+      # Make sure the working tree is clean
+      - run: git reset --hard HEAD
+      - run: |
+          npx lerna publish prepatch \
```

Comment:
Do these commands actually have permissions to publish to NPM? I'd have expected to need to propagate an NPM_TOKEN secret or similar

Reply:
(and probably a GH_TOKEN with extended permissions for `action-gh-release`)

Reply:
The `action-gh-release` uses the GHA token by default. I added a secret (`NPM_TOKEN`) and am now using it.